pkg/operator/daemonset.go

```diff
@@ -70,6 +70,7 @@ func MakeDaemonSet(fb v1alpha2.FluentBit, logPath string) appsv1.DaemonSet {
 					Labels:    fb.Labels,
 				},
 				Spec: corev1.PodSpec{
+					RuntimeClassName:   &fb.Spec.RuntimeClassName,
```

Comment:
It needs to judge whether the `fb.Spec.RuntimeClassName` is "".

If the `fb.Spec.RuntimeClassName` is "", an `dns1123SubdomainError` will be occured.

Reply:
Good catch ; I had assumed since empty was the default it would allow setting the empty string

Should have checked 🤦